### PR TITLE
Add development workflow.

### PR DESCRIPTION
**Pull request by ioquatix**

### CODE REVIEW

**Comment by ioquatix** [comment]
@eregon know you are busy but we have some truffleruby failures. If you have time to check it sometime over the next week that would be awesome.

**Comment by eregon** [comment]
The error is from EventMachine:
https://github.com/rack/rack/runs/429661789
```
./eventmachine.h:131:6: note: candidate function not viable: cannot convert
argument of incomplete type 'VALUE' (aka 'void *') to 'int' for 1st argument
        int evma_set_rlimit_nofile (int n_files);
            ^
4 warnings and 2 errors generated.
```

I fixed this in EventMachine a year ago: https://github.com/eventmachine/eventmachine/pull/878
But there wasn't a release since then.
Maybe do you know a better way to contact the maintainer @sodabrew ?

As a workaround until then, maybe we can skip installing EventMachine on TruffleRuby?

**Comment by sodabrew** [comment]
Nudge for EM release received.

**Comment by eregon** [comment]
@sodabrew Great :)

**Comment by eregon** [comment]
For the record, here are the logs of the test suite on TruffleRuby:
https://github.com/rack/rack/runs/434161173
https://github.com/rack/rack/runs/434161209

File: Gemfile [hunk 1]
```diff
@@ -4,24 +4,12 @@ source 'https://rubygems.org'
...
-# Alternative solution that might work, but it has bad interactions with
-# Gemfile.lock if that gets committed/reused:
-# c_platforms = [:mri] if Gem.platforms.last.os == "java"
-
 group :extra do
-  gem 'fcgi', platforms: c_platforms
+  gem 'fcgi', platform: :mri
   gem 'dalli'
-  gem 'thin', platforms: c_platforms
+  gem 'thin', platform: :mri
```
Comment (eregon) [inline Gemfile]: Nice cleanup!

**Pull request merged**